src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProcessor.cs

```diff
@@ -35,59 +73,130 @@ public ConsoleLoggerProcessor(IConsole console, IConsole errorConsole)
 
         public virtual void EnqueueMessage(LogMessageEntry message)
         {
-            if (!_messageQueue.IsAddingCompleted)
-            {
-                try
-                {
-                    _messageQueue.Add(message);
-                    return;
-                }
-                catch (InvalidOperationException) { }
-            }
-
-            // Adding is completed so just log the message
-            try
+            // cannot enqueue when adding is completed
+            if (!Enqueue(message))
             {
                 WriteMessage(message);
             }
-            catch (Exception) { }
         }
 
         // for testing
```

Comment:
This method is only for testing, or it's internal rather than private for testing?  Seems like the latter, in which case it'd be good to change it to be:
```suggestion
        // internal for testing
```